apps/test-pololu370-encoder/src/main.cpp

```diff
@@ -0,0 +1,41 @@
+#include "Pololu37D.h"
+
+int main() {
+  // first pin is yellow wire
+  // second pin is white wire
+  constexpr Encoder::Pololu37D::Config config = {PA_5, PA_6, NC, 0, GPIO::QEI::Encoding::X2_ENCODING};
+  Encoder::Pololu37D enc(config);
+
+  Timer theta_timer, theta_dot_timer;
+  float theta     = 0;
+  float theta_dot = 0;
+  while (true) {
+    theta_timer.reset();
+    theta_dot_timer.reset();
+
+    // try reading the encoder
+    if (enc.read()) {
+      // angle reading
+      theta_timer.start();
+      if (enc.getAngleDeg(theta)) {
+        theta_timer.stop();
+        printf("Angle reading took: %lluus\tAngle: %.3f\r\n", theta_timer.elapsed_time().count(), theta);
```

Comment:
Can you replace all the printfs with `Utility::Logger::printf()`? We're transitioning to the Logger class, so you'll need to include that library (see apps/test-logger for an example)

Reply:
done

Reply:
i replaced them but they were not printing messages properly

Reply:
Hmmm, like nothing gets printed at all or the prints don't look right?

Reply:
nothing gets printed out

Reply:
Can you try `make clean` and then rebuild the app? See if that fixes it

Reply:
they are working now, make clean fixed the issue.